holepunch/__init__.py

```diff
@@ -310,22 +310,32 @@ def holepunch(args):
                     group=group,
                     ip_permissions=to_remove)
 
+    # Make sure we have a chance to clean up the security group
+    # rules gracefully by ignoring common signals.
+    def terminate(num, frame):
+        print("\nTerminate signal received.")
+
+        # Terminating gracefully subprocess, so it will not screw parent shell
+        if command is not None:
+            process.terminate()
+
+        sys.exit(1)
```

Comment:
What do you think is the right exit code should be?
There are two scenarios:
1. if `holepunch` (i.e. python) is killed then exit code non-zero
2. if `holepunch` without command stopped by Ctrl+C then zero or non-zero?